agent-inject/agent/agent.go

```diff
@@ -246,9 +246,14 @@ type Vault struct {
 	AuthConfig map[string]interface{}
 
 	// CACert is the name of the Certificate Authority certificate
-	// to use when validating Vault's server certificates.
+	// to use when validating Vault's server certificates. It takes
+	// precedence over CACertBytes.
 	CACert string
 
+	// CACertBytes is the contents of the CA certificate to trust
+	// for TLS with Vault as a PEM-encoded certificate or bundle.
```

Comment:
It can be PEM-encoded or base64. I added some comments in https://github.com/hashicorp/vault-k8s/pull/507/commits/d50e787e0e4cb65f43ef44bb8498d69585a2ea95 to document the optional base64 encoding.